subprojects/ide-native/src/integTest/groovy/org/gradle/ide/xcode/fixtures/AbstractXcodeIntegrationSpec.groovy

```diff
@@ -51,4 +51,8 @@ rootProject.name = "${rootProjectName}"
     protected XcodeWorkspacePackage xcodeWorkspace(String path) {
         new XcodeWorkspacePackage(file(path))
     }
+
+    protected XcodebuildExecuter newXcodebuildExecuter() {
```

Comment:
As a nit, I think this might look better as `xcodebuild()`

Reply:
I changed it.